autometa/common/external/samtools.py

```diff
@@ -51,24 +51,29 @@ def sort(sam, bam, nproc=mp.cpu_count()):
     
     """
 
-    cmd = f'samtools view -@{nproc} -bS {sam} | samtools sort -@{nproc} -o {bam}'
+    samtools_out_dir = os.path.dirname(os.path.abspath(bam))
+    outfile_tail = os.path.basename(bam)
```

Comment:
What is the reasoning for the tail? I'm a bit confused with the variable name here.

Reply:
It indicates the file name. Like, ``/home/siddharth/Downloads/align.bam`` tail was supposed to indicate ``align.bam``. I could replace ``tail`` everywhere with ``name`` is that more appropriate?